.github/workflows/build.sh

```diff
@@ -40,6 +40,7 @@ mdbook build -d "$dest_dir"
 
 # Disable the redbox button in built versions of the course
 echo '// Disabled in published builds, see build.sh' > "${dest_dir}/html/theme/redbox.js"
+echo '// Disabled in published builds, see build.sh' > "${dest_dir}/html/theme/instructor-menu.js"
```

Comment:
Oh! I think we don't want this anymore, as we will need the "reset playgrounds" option in the published build. @mgeisler now that we have all of this in a sub-menu, what do you think?